test/e2e/utils/kubernetes.go

```diff
@@ -486,6 +486,38 @@ func (k TestKubernetes) WaitForExternalService(ispn *ispnv1.Infinispan, timeout
 	return hostAndPort
 }
 
+// GetPVC returns a PVC bound with given PVC name
+func (k TestKubernetes) GetPVC(pvcName, namespace string) *corev1.PersistentVolumeClaim {
+	pvc := &corev1.PersistentVolumeClaim{}
+	err := k.Kubernetes.Client.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: pvcName}, pvc)
+	ExpectNoError(err)
+	return pvc
+}
+
+// GetSCList returns the default StorageClasses name in cluster.
+func (k TestKubernetes) GetSCList() string {
+	var defaultStorageClassName string
+
+	// Get the list of StorageClasses in cluster.
+	scList := &storagev1.StorageClassList{}
+	listOps := &client.ListOptions{}
+	err := k.Kubernetes.Client.List(context.TODO(), scList, listOps)
```

Comment:
As `ListOps` is never used again, I would just do: `err := k.Kubernetes.Client.List(context.TODO(), scList,  &client.ListOptions{})`